t/admin/script.t

```diff
@@ -0,0 +1,103 @@
+#
```

Comment:
Can we add this under `t/script` instead?

Reply:
got